### PR TITLE
Combine regex for ocs-provider and ocm-provider

### PR DESCRIPTION
**Pull request by josh4trunks**
Combine regex for ocs-provider and ocm-provider. Slightly improves https://github.com/nextcloud/documentation/commit/b714250b2c3288f29f8a9132739db2de1d2845ce#diff-490ed055ec68f0382edb4b4d680b761b by @schiessle 

If someone could test this change to confirm I would appreciate it. Thank you

### CODE REVIEW

File: admin_manual/installation/nginx.rst [hunk 1]
```diff
@@ -3,7 +3,8 @@ Nginx configuration
 ===================
 
 This page covers example Nginx configurations to use with running a Nextcloud
-server. This page is community-maintained. (Thank you, contributors!)
+server. These configurations examples were originaly provided by
```
Comment (MorrisJobke) [inline admin_manual/installation/nginx.rst]:
```suggestion
server. These configurations examples were originally provided by
```

Comment (josh4trunks) [reply admin_manual/installation/nginx.rst]: Thanks spelling is hard.

**Comment by josh4trunks** [comment]
I would appreciate review of this PR. It is very minor and should be quick to review/test.
Thanks!

**Pull request merged**